eth_account/_utils/legacy_transactions.py

```diff
@@ -141,39 +177,18 @@ def chain_id_to_v(transaction_dict):
 
 
 @curry
-def fill_transaction_defaults(transaction):
-    return merge(TRANSACTION_DEFAULTS, transaction)
-
-
-UNSIGNED_TRANSACTION_FIELDS = (
-    ("nonce", big_endian_int),
-    ("gasPrice", big_endian_int),
-    ("gas", big_endian_int),
-    ("to", Binary.fixed_length(20, allow_empty=True)),
-    ("value", big_endian_int),
-    ("data", binary),
-)
-
-
-class Transaction(HashableRLP):
-    fields = UNSIGNED_TRANSACTION_FIELDS + (
-        ("v", big_endian_int),
-        ("r", big_endian_int),
-        ("s", big_endian_int),
-    )
-
-
-class UnsignedTransaction(HashableRLP):
-    fields = UNSIGNED_TRANSACTION_FIELDS
+def fill_transaction_defaults(transaction_dict: Dict[str, Any]) -> Dict[str, Any]:
+    # type ignored because merge does not preserve typing
+    return merge(TRANSACTION_DEFAULTS, transaction_dict)  # type: ignore[no-any-return]
```

Comment:
Can we cast this to `UnsignedTransaction` instead of ignoring?

Reply:
Is now new `TransactionDictType`